man/man1/parameter_files.1

```diff
@@ -0,0 +1,14 @@
+ .\" Man page for parameter_files
+.TH man 1 "26 November 2020" "1.0" "parameter_files man page"
+.SH NAME
+parameter_files  \- This script will fetch all input files from a configuration file.
```

Comment:
I think a better description for this script is `This script will fetch all parameter files from a given configuration file.`